_posts/2020-07-15-june-rubygems-update.md

```diff
@@ -0,0 +1,53 @@
+---
+title: June 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in June.
+
+
+## RubyGems News
+
+In June, RubyGems saw significant improvements on Bundler and RubyGems development. We worked on synchronizing the latest versions of both libraries with ruby-core, and fixed issues that came up. We also [enforced our ruby-core workflow](https://github.com/rubygems/rubygems/pull/3725) to try to prevent further tedious integrations with ruby-core in the future by catching issues early and minimizing the back and forth of patches between upstream and downstream.
+
+We changed [bundler specs to raise by default when any subcommand fails](https://github.com/rubygems/rubygems/pull/3685). This action helped reveal two bugs (which we of course, fixed!). We've adapted bundler release tasks to a [new repository layout](https://github.com/rubygems/rubygems/pull/3703) and now have the changelog draft up-to-date.
+
+
+We've also implemented a fix to [stop soft-validating gemspecs](https://github.com/rubygems/rubygems/pull/3668) (e.g. giving validations that only warn) except for in gem-authoring contexts. We also added a [slack notification to the maintainers channel](https://github.com/rubygems/rubygems/pull/3689) that triggers whenever the  `ruby-head` builds starts failing.
+
+We [simplified our CI Workflows](https://github.com/rubygems/rubygems/pull/3769) and did some final cleanup PRs related to CI failures on `jruby`.
+
+Lastly, onto great news for the month: we released Bundler 2.2.0.rc.1! 🎉
+
+Outside of these major highlights, we continue to fix bugs, review PRs, improve our documentation, cleanup test suites, and continue our usual ongoing maintenance.
+
+In total, [RubyGems](https://github.com/rubygems) gained [243 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2020-06-01%7D...master@%7B2020-06-30%7D), contributed by 15 authors. There were 3003 additions and 2432 deletions across 313 files.
+
+## RubyGems.org News
+
+This month, we added terraform module for RDS monitors and added alerts on `rubgems-production`, `rubygems-staging` and `shipit` instances. We followed up with fixes on RubyGems.org that have now been deployed and verified! We also debugged failed delayed jobs in production and created a [fix for issues we found with regex validation with user email.](https://github.com/rubygems/rubygems.org/pull/2389)
```

Comment:
> fix for **issues** we found

Perhaps add following two as well since you are using issues. These are related to failed delayed job. 
https://github.com/rubygems/rubygems.org/pull/2406
https://github.com/rubygems/rubygems.org/pull/2388

Reply:
Sure thanks @sonalkr132